src/main/java/duke/exceptions/TaskModificationException.java

```diff
@@ -0,0 +1,18 @@
+package duke.exceptions;
+
+/**
+ * This class implements an excpetion for when the bot fails to modify a task.
+ *
+ * @author delishad21
+ */
+public class TaskModificationException extends DukeException {
+
+    /**
+     * Creates a TaskModificationException, thrown when user tries to modify a task but fails to.
+     *
+     * @param errorMsg Reason for task modification failure.
+     */
+    public TaskModificationException(String errorMsg) {
+        super(errorMsg);
```

Comment:
Would changing `errorMsg` to `errorMessage` be better for easier pronunciation and readability?